book/src/design/circuit/gadgets/sinsemilla.md

```diff
@@ -138,16 +138,19 @@ y_{A,i+1} &= (\lambda_{1,i+1} + \lambda_{2,i+1}) \cdot (x_{A,i+1} - (\lambda_{1,
 \end{aligned}
 $$
 
+* The degree of a lookup gate is 2 + the degree of the polynomial expression being looked up (after tuple compression). TODO check this.
```

Comment:
A blank line night be needed here for correct formatting (I forgot to add one in my suggested change).